### DISCUSSION TITLE
How to use PrivateKeyModule in tKey

### DISCUSSION THREAD

**Discussion by aristidesgpg** [Web3Auth/Web3Auth#759] [Help]
Could you tell me how I can initialize PrivateKeyModule?
In the document, it says like this,
`const privateKeyModule = new PrivateKeyModule();`
But , it requires `IPrivateKeyFormat`, not sure how it can be initialized

**Comment by BboyStatix** [comment]
Hey there @talentdev2020 thanks for pointing this out. We've updated the docs here https://web3auth.io/docs/sdk/self-host/modules/private-keys#assign-the-privatekeymodule-class-to-a-variable

Feel free to message again if it's still not clear.
